app/store/index.js

```diff
@@ -1,11 +1,17 @@
-import {createStore, applyMiddleware} from 'redux';
-import rootReducer from '../reducers';
-import thunk from 'redux-thunk';
+import { createStore, applyMiddleware, compose } from "redux";
+import rootReducer from "../reducers";
+import thunk from "redux-thunk";
+import { createLogger } from "redux-logger";
 
 export default function configureStore(initialState) {
-  return createStore(
-    rootReducer,
-    initialState,
-    applyMiddleware(thunk)
-  );
-}
\ No newline at end of file
+  const debugware = [];
```

Comment:
Do we really need this here? if you want it for dev debugging then create a production store that doesn't have it.